<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>471</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="0" colspan="3" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QLabel" name="l_csvpath">
      <property name="text">
       <string>Import CSV file containing contact numbers.</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pb_savetoken">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: black;
	border-radius: 10px;
	font-weight: bold;
	font-size: 16px;
	padding: 0 0 5px 0;
	color:white;
}
QPushButton:pressed {
	background-color: rgb(0, 255, 255);
}
</string>
      </property>
      <property name="text">
       <string>Save Current Token</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="le_watoken">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="pb_import">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Import CSV</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" alignment="Qt::AlignRight">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Group List</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="cb_grouplist">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0" alignment="Qt::AlignRight">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Whapi.cloud Token</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pb_sync">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgb(0, 212, 0);
	border-radius: 10px;
	font-weight: bold;
	font-size: 16px;
	padding: 0 0 5px 0;
}
QPushButton:pressed {
	background-color: rgb(0, 255, 127);
}
QPushButton:disabled {
	background-color: rgb(85, 255, 255);
}
</string>
      </property>
      <property name="text">
       <string>Sync</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="2" alignment="Qt::AlignRight">
     <widget class="QLabel" name="l_req_left">
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Req. left: -</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>230</y>
     <width>471</width>
     <height>361</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>469</width>
      <height>359</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
